modules/widgets/google-translate.php

```diff
@@ -66,7 +66,27 @@ public function widget( $args, $instance ) {
 				'title' => $this->default_title,
 			) );
 
-			wp_localize_script( 'google-translate-init', '_wp_google_translate_widget', array( 'lang' => get_locale() ) );
+			wp_localize_script(
+				'google-translate-init',
+				'_wp_google_translate_widget',
+				array(
+					'lang'   => get_locale(),
+					/**
+					 * Filter the layout of the Google Translate Widget.
+					 *
+					 * 3 different values are accepted. Nothing, google.translate.TranslateElement.InlineLayout.SIMPLE, or google.translate.TranslateElement.InlineLayout.HORIZONTAL
+					 *
+					 * @see https://translate.google.com/manager/website/
+					 *
+					 * @module widgets
+					 *
+					 * @since 5.1.0
+					 *
+					 * @param string $layout layout of the Google Translate Widget.
+					 */
+					'layout' => esc_js( apply_filters( 'jetpack_google_translate_widget_layout', '' ) ),
```

Comment:
On one hand, this is incompatible with what `google.translate.TranslateElement` accepts for the `layout` property, it accepts a number, not a string.
Logging in console:
`google.translate.TranslateElement.InlineLayout.VERTICAL` throws `0`
`google.translate.TranslateElement.InlineLayout.HORIZONTAL` throws `1`
`google.translate.TranslateElement.InlineLayout.SIMPLE` throws `2`
So this filter should accept a number and validate that number. Otherwise a method should be devised in JS to parse the route received as string and resolve it in the `google` object to get the right value.

However, this is very cumbersome for users. I'd rather have a `<select>` controls that user can pull to choose the desired layout.

Reply:
That makes sense. I don't know that we need a UI for this though; we don't really receive requests for that option.

I updated the PR to validate the filter and only accept integers between 0 and 2. If anything else, we fallback to the default.